spec/support/factory_girl.rb

```diff
@@ -9,7 +9,7 @@
   end
 
   factory :food_truck do
-    sequence(:name) { |n| "foodtruck#{n}" }
+    sequence(:name) { |m| "foodtruck#{m}" }
```

Comment:
Just FYI: No need to change this variable name, it's in a block, isolated from the sequence for the header.